application/FtbListModel.cpp

```diff
@@ -70,14 +100,13 @@ int FtbListModel::columnCount(const QModelIndex &parent) const
 QVariant FtbListModel::data(const QModelIndex &index, int role) const
 {
 	int pos = index.row();
-	if(modpacks.size() <= pos || pos < 0) {
+	if(modpacks.size() <= pos || pos < 0 || !index.isValid()) {
```

Comment:
The existing code would consider index one past the end valid.
Use: `pos >= modpacks.size() || pos < 0 || !index.isValid()`